### PR TITLE
Added information about enrollment and verification with Duo as second …

### PR DESCRIPTION
**Pull request by nikolaystoilov-okta**
…factor using the API.

Resolves: OKTA-92692

### CODE REVIEW

**Comment by nikolaystoilov-okta** [comment]
@mystiberry-okta @karlmcguinness-okta @slee-okta @joelfranusic-okta @srinivasanagandla-okta 
please review.

**Comment by nikolaystoilov-okta** [comment]
@nandagopalseshagiri-okta @madhavimanduri-okta please review.

File: _source/docs/api/resources/authn.md [hunk 1]
```diff
@@ -1915,6 +1916,235 @@ curl -v -X POST \
...
+##### Launch Duo WebSdk iFrame
+{:.api .api-response .api-response-example}
+
+~~~html
+<script src="/js/sections/duo/Duo-Web-v2.js"></script>
+<script>
+  Duo.init({
+    'host': '{activation.host}',
+    'sig_request': '{activation.signature}',
+    'post_action': '{activation._links.complete.href}'
```
Comment (nandagopalseshagiri-okta) [inline _source/docs/api/resources/authn.md]: where would one get the values for activation.host, signature and complete.href

File: _source/docs/api/resources/authn.md [hunk 2]
```diff
@@ -1919,7 +1919,7 @@ curl -v -X POST \
 #### Enroll Duo Factor
 {:.api .api-operation}
 
-Enrolls a user with a Duo factor.  The enrollment process is interaction between OKTA and Duo code. The process starts with enrollment request to OKTA and then continue with Dou widget/script that handles the enrollment. At the end of the process it will make a callback to Okta to complete the enrollment.
+Enrolls a user with a Duo factor.  The enrollment process starts with an enrollment request to Okta, then continues with a Duo widget or script that handles the enrollment. At the end of the process, Duo makes a callback to Okta to complete the enrollment.
```
Comment (nikolaystoilov-okta) [inline _source/docs/api/resources/authn.md]: "then continues with a Duo widget or script that handles the enrollment" should be
"then continues with a Duo widget **and** script that handles the enrollment"

**Pull request merged**